MapboxCoreNavigation/CLLocation.swift

```diff
@@ -10,8 +10,7 @@ extension CLLocation {
         #else
             return
                 0...100 ~= horizontalAccuracy &&
-                verticalAccuracy > 0 &&
-                speed >= 0
+                0...30 ~= horizontalAccuracy
```

Comment:
Something went wrong here. Between 0-100 and between 0-30 for the same horizontal accuracy.

Reply:
oof my bad, will fix.